x-pack/plugins/ingest_manager/server/services/epm/config.ts

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I think we can skip this file or much of it. The Joi schema stuff is handled by Ingest and the config store is handled by the config service (`server/services/config.ts`)

Reply:
Or if we want to keep the information about config here, we can still export `config` and import/use that value in https://github.com/elastic/kibana/blob/34bceeddcd216dae9fa5786a5423a805758c8ce1/x-pack/plugins/ingest_manager/server/index.ts#L18-L21